### PR TITLE
Temporarily disable mail tutorial.

### PR DESCRIPTION
**Pull request by Unengine**
### Description

1. Temporarily disable mail tutorial.

### How to test

1. Delete all `PlayerPrefs` and create a new character.
2. Check if the following tutorial is removed.
    - Receiving crafted equipment in mail.
    - Equipping received equipment.

### Required Reviewers
@planetarium/9c-dev and @Namyujeong

### CODE REVIEW

File: nekoyume/Assets/_Scripts/UI/Mail.cs [hunk 1]
```diff
@@ -178,22 +178,23 @@ private void UpdateMailList(long blockIndex)
...
-                mail.requiredBlockIndex <= blockIndex &&
-                !(mail.attachment.itemUsable is null) &&
-                mail.attachment.itemUsable.ItemSubType == Nekoyume.Model.Item.ItemSubType.Weapon &&
-                mail.attachment.itemUsable.Id == TutorialEquipmentId);
-                if (!(tutorialEquipment is null))
-                {
-                    OnReceivedTutorialEquipment();
-                }
-            }
+            // Temporarily disable mail tutorial.
```
Comment (hskim881028) [inline nekoyume/Assets/_Scripts/UI/Mail.cs:181]: Why did you comment out the code without deleting it?

File: nekoyume/Assets/_Scripts/UI/Menu.cs [hunk 2]
```diff
@@ -444,8 +444,10 @@ private void PlayTutorial()
                 if (recipeRow is null)
                 {
                     Debug.LogError("EquipmentItemRecipeSheet is empty");
+                    return;
                 }
-                else if (States.Instance.CurrentAvatarState.inventory.HasItem(recipeRow.ResultEquipmentId))
+
+                if (!States.Instance.CurrentAvatarState.inventory.HasItem(recipeRow.MaterialId))
```
Comment (hskim881028) [inline nekoyume/Assets/_Scripts/UI/Menu.cs:450]: if `recipeRow` is null, it will be throw exception

Comment (Unengine) [reply nekoyume/Assets/_Scripts/UI/Menu.cs:450]:
```
if (recipeRow is null)
{
    Debug.LogError("EquipmentItemRecipeSheet is empty");
    return;
}
```
`recipeRow` variable is already filtered at this moment.

File: nekoyume/Assets/_Scripts/UI/Mail.cs (hunk 1 above)
Comment (Unengine) [reply nekoyume/Assets/_Scripts/UI/Mail.cs:181]: This code might be rollbacked later, but I think it would rather deleting it.

**Review by Namyujeong — APPROVED** [review]
- Reduced unnecessary click points.
- Regardless of whether or not the equipment is completed, it is possible to proceed with the next step.

**Review by hskim881028 — APPROVED** [review]

**Review by boscohyun — COMMENTED** [review]
Why was the `nekoyume/Assets/Resources/Font/SDF/English SDF.asset` file changed?

**Comment by Unengine** [comment]
> Why was the `nekoyume/Assets/Resources/Font/SDF/English SDF.asset` file changed?

This is unexpected. I will fix it ASAP!

**Review by boscohyun — APPROVED** [review]

**Pull request merged**